### PR TITLE
Display wind speed values on the map when the mouse is over a region.

### PR DESCRIPTION
**Pull request by 0nza1101**
-Created L.ControlVelocity.ts wich is used to see the wind speed.
-Added leaflet-velocity.css to src (need a better implementation).
-Added interpolate and bilinearInterpolateVector and also isValue, floorMod to windy.ts.
-Added initMouseHandler to L.VelocityLayer.ts used to listen mouseEvent and display wind speed value.

### CODE REVIEW

File: src/L.ControlVelocity.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  meterSec2Knots(meters: number) {
+    return meters / 0.514
+  }
+
+  meterSec2kilometerHour(meters: number) {
+    return meters * 3.6
+  }
+
+  _drawWindSpeed(ev: any) {
+    var pos = this._map.containerPointToLatLng(L.point(ev.containerPoint.x, ev.containerPoint.y));
```
Comment (landru29) [inline src/L.ControlVelocity.ts]: Not ts
Use let or const

File: src/L.ControlVelocity.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    var gridValue = this._windy.interpolate(pos.lng, pos.lat);
+    var htmlOut = "";
+    console.log("Grid value ", gridValue);
```
Comment (landru29) [inline src/L.ControlVelocity.ts]: No console.log

File: src/L.ControlVelocity.ts [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    if (gridValue && !isNaN(gridValue[0]) && !isNaN(gridValue[1]) && gridValue[2]) {
+      htmlOut = "<strong>  Direction: </strong>" +
```
Comment (landru29) [inline src/L.ControlVelocity.ts]: Use template

File: src/L.ControlVelocity.ts [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        this.vectorToDegrees(gridValue[0], gridValue[1], this.options.angleConvention).toFixed(3) +
+        "°" + ", <thisstrong>  Speed: </strong>" +
+        this.vectorToSpeed(gridValue[0], gridValue[1], "m/s").toFixed(1)+ " m/s";
+    }
+    else {
+      htmlOut = this.options.displayOptions.emptyString;
+    }
+    this._container.innerHTML = htmlOut;
+    /* move control to bottom row
```
Comment (landru29) [inline src/L.ControlVelocity.ts]: Unused?
Delete

File: src/L.VelocityLayer.ts [hunk 5]
```diff
@@ -1,153 +1,176 @@
...
+import CanvasBound from './canvasBound'
+import MapBound from './mapBound';
+import Layer from "./layer";
+import CanvasLayer from './L.CanvasLayer';
+import VelocityControl from './L.ControlVelocity'
+import * as L from 'leaflet';
+
+
+const L_CanvasLayer = (L.Layer ? L.Layer : L.Class).extend(new CanvasLayer());
+let L_canvasLayer = function() {
```
Comment (landru29) [inline src/L.VelocityLayer.ts]: Const or let@

File: src/L.VelocityLayer.ts [hunk 6]
```diff
@@ -1,153 +1,176 @@
...
+    });
+
+    this._initMouseHandler();
+  }
+
+  _initMouseHandler() {
+    if (!this._mouseControl && this.options.displayValues) {
+      var options = this.options.displayOptions || {};
+      options['leafletVelocity'] = this;
+      this._mouseControl = L_controlVelocity().delegate(this);
```
Comment (landru29) [inline src/L.VelocityLayer.ts]: I think, the method "delegate" should be removed

**Review by landru29 — COMMENTED** [review]
Too difficult to review as you changed indentation

**Comment by 0nza1101** [comment]
It should be good now let me know if it's ok.

**Pull request closed without merging**